ci-operator/step-registry/telco5g/ptp/tests/telco5g-ptp-tests-commands.sh

```diff
@@ -68,8 +71,16 @@ spec:
           # set +x here to hide pass from log
           set +xe
           echo "podman login with serviceaccount"
-          pass=$( jq .\"image-registry.openshift-image-registry.svc:5000\".password /var/run/secrets/openshift.io/push/.dockercfg )
-          podman login -u serviceaccount -p ${pass:1:-1} image-registry.openshift-image-registry.svc:5000 --tls-verify=false
+
+          if [[ "$T5CI_VERSION" == "4.16" || "$T5CI_VERSION" == "4.17"]];
```

Comment:
I would check `if version > 4.15 instead`. That way we don't have to update this with every future release.

Reply:
I didnt want to use the release version in this logic so I have updated to try the old way first and using a try catch block to detect failure and use the logic as a catch for newer releases.